assets/src/design-system/images/index.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+export { ReactComponent as WebStoriesLogo } from './webStoriesFullLogo.svg';
```

Comment:
I'm torn about this being here? We've been trying to keep images in assets/images and i can't remember why this was specifically in dashboard/images instead.

Reply:
I can put it back and we can always move it later 👍